wire-schema/src/jvmMain/resources/google/protobuf/descriptor.proto

```diff
@@ -854,6 +889,99 @@ message UninterpretedOption {
   optional string aggregate_value = 8;
 }
 
+// ===================================================================
+// Features
+
+// TODO(b/274655146) Enums in C++ gencode (and potentially other languages) are
+// not well scoped.  This means that each of the feature enums below can clash
+// with each other.  The short names we've chosen maximize call-site
+// readability, but leave us very open to this scenario.  A future feature will
+// be designed and implemented to handle this, hopefully before we ever hit a
+// conflict here.
+message FeatureSet {
```

Comment:
If we ever wanna support editions when generating code, that's the logic we need to check